config/jenkins/README.md

```diff
@@ -1,212 +1,53 @@
-        tmpfs           6.9G     0  6.9G   0% /sys/fs/cgroup
-        /dev/sdb15      105M  3.6M  101M   4% /boot/efi
-        /dev/sdc1        28G   45M   26G   1% /mnt
-        tmpfs           1.4G     0  1.4G   0% /run/user/998
-        tmpfs           1.4G     0  1.4G   0% /run/user/1000
-        /dev/sda1       503G   73M  478G   1% /var/jenkins_home
-        ```
-        And it should be mounted automatically at boot time.
-
-
-8. Clone the openenclave-infra-config repository and `cd` to it:
-    ```
-    git clone https://github.com/openenclave-ci/test-infra
-    cd test-infra/config/jenkins/master
-    ```
-
-9. Run the Jenkins master setup script under `root` user:
-    ```
-    sudo su
-    export JENKINS_PUBLIC_DNS_ADDRESS=oe-prow-testing.uksouth.cloudapp.azure.com
-    ./setup.sh
-    ```
-    The setup script does the following:
-    * Docker installation
-    * NGinx installation (used as the HTTPs proxy for the Jenkins master)
-    * Letsencrypt SSL certificates configuration (done via certbot)
-    * Jenkins master systemd configuration
-    * HTTPs proxy for Jenkins master via NGinx configuration
-
-10. (experimatental) Install the saved `plugins.txt` for Jenkins master:
-    ```
-    sudo ./install-plugins.sh
-    ```
-    **NOTE**: Keep in mind that it will restart Jenkins master at the end
-
-
-11. Unlock Jenkins, in your browser go to: https://oe-prow-testing.uksouth.cloudapp.azure.com/
-
-In your terminal grab the password from
-```
-sudo cat /var/jenkins_home/secrets/initialAdminPassword
-```
-
-12. Install Plugins We Care about
-
-```
-Azure Virtual Machine Scale Set
-```
\ No newline at end of file
+# Introduction 
+This is used to setup and manage a scaling Jenkins service using the Azure Kubernetes Service via Azure CLI.
+
+Image: Jenkins LTS
+Ingress: Nginx
+Configuration: Managed by Jenkins Configuration as Code Plugin
+Credentials: Jenkins + Azure Key Vault
+Storage: Azure Persistent Volumes
+
+# Getting Started
+## Installation process
+1. Open aks-cluster-deploy.sh, jenkins-install-plugins.sh, and jenkins-install-configuration.sh scripts and change variables specific to your needs. Detailed instructions are written in-script.
+2. Add executable permissions by running: `chmod +x aks-cluster-deploy.sh jenkins-install-plugins.sh jenkins-install-configuration.sh`
+3. Log in to Azure by running: `az login`
+4. Set up AKS cluster by running: `./aks-cluster-deploy.sh`
+5. Install Jenkins plugins by running: `./jenkins-install-plugins.sh`. When this is complete, a new Kubernetes pod will be started. If the new pod can start Jenkins successfully, which takes about 5 - 10 minutes, the old pod will be terminated.
+6. Install Jenkins configuration by running: `./jenkins-install-configuration.sh`. Configuration can be picked up by restarting Jenkins (during a new Jenkins setup) or by running the 'Master/reload-configuration' job on Jenkins.
+
+## Dependencies
+### Azure resource requirements
```

Comment:
A requirement here would be a preset up image gallery preloaded with the machine images we need. Of course, actually _doing_ that work is out of scope for this PR but we should make mention of it.